lib/core/src/lib/mock/card-view-content.mock.ts

```diff
@@ -0,0 +1,185 @@
+            { name: 'hey', value: 'you' }
+        ],
+        key: 'key-value-pairs',
+        editable: true
+    }),
+    new CardViewSelectItemModel({
+        label: 'CardView Select Item',
+        value: 'one',
+        options$: of([
+            { key: 'one', label: 'One' },
+            { key: 'two', label: 'Two' }
+        ]),
+        key: 'select',
+        editable: true
+    }),
+    new CardViewMapItemModel({
+        label: 'My map',
+        value: new Map([['999', 'My Value']]),
+        key: 'map',
+        default: 'default map value'
+    }),
+    new CardViewTextItemModel({
+        label: 'This is clickable ',
+        value: 'click here',
+        key: 'click',
+        default: 'click here',
+        editable: true,
+        clickable: true,
+        icon: 'close'
+    }),
+    new CardViewArrayItemModel({
+        label: 'CardView Array of items',
+        value: of([
+            { icon: 'directions_bike', value: 'Zlatan' },
+            { icon: 'directions_bike', value: 'Lionel Messi' },
+            { value: 'Mohamed', directions_bike: 'save' },
+            { value: 'Ronaldo' }
+        ]),
+        key: 'array',
+        icon: 'edit',
+        default: 'Empty',
+        noOfItemsToDisplay: 2,
+        editable: true
+    })
+];
+
+export const cardViewUndefinedValues = [
+    new CardViewTextItemModel({
+        label: 'CardView Text Item - Multivalue (chips)',
+        value: undefined,
+        key: 'name',
+        default: undefined,
+        multiline: true,
+        multivalued: true,
+        icon: 'icon',
+        editable: true
+    }),
+    new CardViewDateItemModel({
+        label: 'CardView Date Item - Multivalue (chips)',
+        value: undefined,
+        key: 'date',
+        default: undefined,
+        format: 'shortDate',
+        editable: true,
+        multivalued: true
+    }),
+    new CardViewDatetimeItemModel({
+        label: 'CardView Datetime Item - Multivalue (chips)',
+        value: undefined,
+        key: 'datetime',
+        default: undefined,
+        format: 'short',
+        editable: true,
+        multivalued: true
+    }),
+    new CardViewIntItemModel({
+        label: 'CardView Int Item',
+        value: undefined,
+        key: 'int',
+        default: undefined,
+        editable: true
+    }),
+    new CardViewSelectItemModel({
+        label: 'CardView Select Item',
+        value: undefined,
+        options$: of([
+            { key: 'one', label: 'One' },
+            { key: 'two', label: 'Two' }
+        ]),
+        key: 'select',
+        editable: true
+    }),
+    new CardViewMapItemModel({
+        label: 'My map',
+        value: undefined,
+        key: 'map',
+        default: undefined
+    }),
+    new CardViewTextItemModel({
+        label: 'This is clickable ',
+        value: undefined,
+        key: 'click',
+        default: undefined,
+        editable: true,
+        clickable: true,
+        icon: 'close'
+    })
+];
```

Comment:
I think you need to move this file to a new mock folder, like in @Venrofi example:
 `lib/core/src/lib/card-view/components/card-view/mock/card-view-content.mock.ts`

Reply:
ok, so I will do that